src/components/Template/Template.stories.mdx

```diff
@@ -208,6 +208,9 @@ Note: If you need to render the `SkipNavigation` component, you need to explicit
 render it immediately before the `TemplateAboveHeader` component. This is done
 automatically in the `TemplateAppContainer` component.
 
+Note: All components that go above the main content `TemplateContent` component
```

Comment:
This may need to be amended based on my comment in the PR for [DSD-46](https://github.com/NYPL/nypl-design-system/pull/1098).

Reply:
I'll update it in that PR.